tests/integration/test_types_integration.py

```diff
@@ -212,69 +212,74 @@ def test_time(trino_connection):
         .add_field(
             sql="TIME '23:59:59.999'",
             python=time(23, 59, 59, 999000))
-        # TODO: re-enable when Trino 404 is released
-        # .add_field(
-        #     sql="TIME '23:59:59.9999'",
-        #     python=time(0, 0, 0))  # PARAMETRIC_DATETIME not enabled
-        # .add_field(
-        #     sql="TIME '23:59:59.99999'",
-        #     python=time(0, 0, 0))  # PARAMETRIC_DATETIME not enabled
-        # .add_field(
-        #     sql="TIME '23:59:59.999999'",
-        #     python=time(0, 0, 0))  # PARAMETRIC_DATETIME not enabled
-        # round down
         .add_field(
-            sql="TIME '00:00:00.000100'",
```

Comment:
where did this case go?

Reply:
Do we still need this considering there is no rounding involved here?

Reply:
Good catch, the rounding only happenned when parametric timestamp was not implemented. `00:00:00.000100` has `1` at 4th decimal digit and without parametric timestamp support it gets rounded to `00:00:00`.